cuebot/Dockerfile

```diff
@@ -1,73 +1,36 @@
 # -----------------
 # BUILD
 # -----------------
-FROM centos:7 as build
+FROM gradle:6.0.1-jdk11 as build
 
 # First line after FROM should be unique to avoid --cache-from weirdness.
 RUN echo "Cuebot build stage"
 
-RUN yum install -y \
-      java-1.8.0-openjdk.x86_64 \
-      java-1.8.0-openjdk-devel.x86_64 \
-      libaio \
-      which \
-    && yum clean all
+COPY --chown=gradle:gradle ./cuebot /home/gradle/cuebot/
+COPY --chown=gradle:gradle ./proto /home/gradle/proto/
```

Comment:
Is there a way to just switch to the gradle user permanently at the beginning of the build? I don't think we need root permissions for these remaining steps and it might simplify things?

Reply:
The `COPY` command doesn't respect the `USER` specified, so the `COPY` commands have been left as is but I've added a `USER` command to simplify the later `RUN` commands.